lib/tiled.dart

```diff
@@ -2,79 +2,83 @@ import 'dart:math' as math;
 import 'dart:async';
 import 'dart:ui';
 
+import 'package:xml/xml.dart';
+import 'package:flame/components.dart';
 import 'package:flame/flame.dart';
-import 'package:flame/sprite_batch.dart';
+import 'package:flame/sprite.dart';
 import 'package:flutter/material.dart' show Colors;
-import 'package:tiled/tiled.dart' hide Image;
+import 'package:tiled/tiled.dart';
 
 /// Tiled represents all flips and rotation using three possible flips: horizontal, vertical and diagonal.
-/// This class converts that representation to a simpler one, that uses one angle (with pi/2 steps) and two flips (H or V).
+/// This class converts that representation to a simpler one, that uses one angle (with pi/2 steps) and two shifts (X and Y).
```

Comment:
What does `shifts` mean here? It sounds like it is shifting the position, but it has something to do with flips right?